test/private_assets_copy_directory_to_site_test.rb

```diff
@@ -0,0 +1,66 @@
+# team_hub - Components for creating a team Hub using Jekyll
+#
+# Written in 2015 by Mike Bland (michael.bland@gsa.gov)
+# on behalf of the 18F team, part of the US General Services Administration:
+# https://18f.gsa.gov/
+#
+# To the extent possible under law, the author(s) have dedicated all copyright
+# and related and neighboring rights to this software to the public domain
+# worldwide. This software is distributed without any warranty.
+#
+# You should have received a copy of the CC0 Public Domain Dedication along
+# with this software. If not, see
+# <https://creativecommons.org/publicdomain/zero/1.0/>.
+#
+# @author Mike Bland (michael.bland@gsa.gov)
+
+require_relative "test_helper"
+require_relative "../lib/team_hub/private_assets"
+require_relative "site"
+
+require "minitest/autorun"
+require "test_temp_file_helper"
+
+module TeamHub
+  class PrivateAssetsCopyDirectoryToSiteTest < ::Minitest::Test
+    def setup
+      config = {
+        'source' => ENV['TEST_TMPDIR'],
```

Comment:
I believe that using code in one place to set an environment variable to be used in other parts of the same application is an anti-pattern. What if this value were retrieved via a method like `TestTempFileHelper.test_tmpdir`?